source/tutorials/filtering.rst

```diff
@@ -156,7 +164,72 @@ This syntax also supports negating individual clauses of the ``--p-where`` expre
      --p-where "Subject='subject-1' AND NOT BodySite='gut'" \
      --o-filtered-table subject-1-non-gut-filtered-table.qza
 
-.. note:: Currently, the most common metadata-based filtering of features is based on feature taxonomy, such as filtering all features that are annotated as being in a particular genus. This can currently be achieved using ``filter-features`` if taxonomy is provided in a feature metadata file. We are working on adding more direct support for this functionality, which will be made available in a new method of the ``q2-taxa`` plugin. You can track progress on this `here <https://github.com/qiime2/q2-taxa/issues/40>`_.
+Taxonomy-based filtering of tables and sequences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Taxonomy-based filtering is a very common type of feature-metadata-based filtering, so the ``q2-taxa`` plugin provides the ``filter-table`` method to simplify this process. Filtering can be applied to retain only specific taxa using ``--p-include`` and/or to remove specific taxa using ``--p-exclude``.
+
+Removing a feature if its taxonomic annotation contains some specific text is achieved with the ``--p-exclude`` parameter. For example, ``--p-exclude`` is used here to remove all features annotated as ``mitochondria`` from a table.
+
+.. command-block::
+   qiime taxa filter-table \
+     --i-table table.qza \
+     --i-taxonomy taxonomy.qza \
+     --p-exclude mitochondria \
+     --o-filtered-table table-no-mitochondria.qza
+
+Removing features that match more than one search term is achieved by providing the search terms in a comma-separated list. The following command will remove all features that contain either ``mitochondria`` or ``Chloroplast`` in their taxonomic annotation.
```

Comment:
This might be a good point to mention case sensitivity

Reply:
Turns out that ``LIKE`` isn't case sensitive. Added a note on when the queries are (``mode == 'exact'``) and are not (``mode == 'contains'``) case sensitive.